### DISCUSSION TITLE
SDK Installation  Error

### DISCUSSION THREAD

**Discussion by pengbin2015** [aws-solutions/workload-discovery-on-aws#565] [Q&A] [answered]
try to install the JavaScript SDK by the command `yarn add @aws/workload-discovery-sdk`. It seems the package is not available.

➤ YN0027: @aws/workload-discovery-sdk@unknown can't be resolved to a satisfying range
➤ YN0035: Package not found
➤ YN0035:   Response Code: 404 (Not Found)
➤ YN0035:   Request Method: GET
➤ YN0035:   Request URL: https://registry.yarnpkg.com/@aws%2fworkload-discovery-sdk

➤ Errors happened when preparing the environment required to run this command.

**Comment by svozza** [comment, marked as answer]
Apologies, the SDK is not on NPM unfortunately. That documentation in the SDK section is not correct and the SDK code was meant is also out of date. I will add a backlog item to fix this in an upcoming patch release.

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
